contracts/src/bridge/Bridge.sol

```diff
@@ -32,7 +32,7 @@ contract Bridge is OwnableUpgradeable, DelegateCallAware, IBridge {
     address[] public allowedInboxList;
     address[] public allowedOutboxList;
 
-    address public override activeOutbox;
+    address public override activeOutbox = address(type(uint160).max);
```

Comment:
We don't want to make this a breaking change, so we can manually make a `activeOutbox` getter
```suggestion
    address private _activeOutbox = address(type(uint160).max);
    
    function activeOutbox() public view returns (address) { ... }
```

Also, assigning to `activeOutbox` like this, means it gets run in the constructor. 
We should add this to the `initialize` function